tensorflow-notebook/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+FROM jupyter/scipy-notebook
+
+MAINTAINER Jupyter Project <jupyter@googlegroups.com>
+
+USER $NB_USER
+
+# Install Python 3 Tensorflow
+RUN conda install --quiet --yes -c conda-forge 'tensorflow=0.9.0'
```

Comment:
conda-forge is in the default channels list in the base-notebook image. You can exclude it here.